src/main/cluster-manager.ts

```diff
@@ -14,7 +14,7 @@ export class ClusterManager extends Singleton {
     // auto-init clusters
     autorun(() => {
       clusterStore.enabledClustersList.forEach(cluster => {
-        if (!cluster.initialized) {
+        if (!cluster.initialized && !cluster.initializing) {
```

Comment:
Can't get why only `initialized` not enough? They are both called in `cluster.init()` after some `await`s..